core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.html

```diff
@@ -24,7 +24,9 @@ <h2 class="tab-title" *ngIf="displayArea === 'topicViewer'">
       </mat-card-content>
       <mat-card-actions class="card-actions">
         <div class="fx-row fx-main-center fx-cross-center full-width">
-          <button mat-raised-button [disabled]="isStartButtonDisabled()" (click)="openNewPracticeSession()" class="practice-session-button protractor-test-practice-start-button">
+          <button mat-raised-button [disabled]="isStartButtonDisabled() && questionsStatusCallIsComplete"
```

Comment:
I don't think we need to check questionsStatusCallIsComplete here because we check it below in the ngClass.

Reply:
`questionsStatusCallIsComplete` is used in ngClass just to provide styling(background) to button on loading.
whereas `isStartButtonDisabled() && questionsStatusCallIsComplete` is used in [disabled] to actually disable the button when questions are loading and when no checkboxs are selected.

Reply:
Done!